src/main/java/eu/deltasource/internship/abankingsystem/model/Owner.java

```diff
@@ -0,0 +1,49 @@
+package eu.deltasource.internship.abankingsystem.model;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Owner {
+
+    private int id;
+
+    private final String name;
+
+    private final List<BankAccount> ownerAccountCount = new ArrayList<>();
+
+    private final List<BankAccount> bankAccounts = new ArrayList<>();
+
+    public Owner(String name) {
+        this.name = name;
+    }
+
+    public Owner(int id, String name) {
+        this.id = id;
+        this.name = name;
+    }
+
+    public List<BankAccount> getOwnerAccountCount() {
+        return Collections.unmodifiableList(ownerAccountCount);
+    }
+
+    public void addAccountToOwner(final BankAccount bankAccount) {
```

Comment:
For this method it would really be better to make a null check. 
I am using final parameters in a method to prevent the value of the parameter to be changed inside of the body of the method.
This is more like a developer's protection not to make mistakes in the code.